Mathlib/Algebra/MvPolynomial/Equiv.lean

```diff
@@ -441,7 +440,7 @@ lemma totalDegree_coeff_finSuccEquiv_add_le (f : MvPolynomial (Fin (n + 1)) R) (
   · rw [← support_coeff_finSuccEquiv]
     exact hσ1
 
-theorem finSuccEquiv_support (f : MvPolynomial (Fin (n + 1)) R) :
+theorem support_finSuccEquiv {f : MvPolynomial (Fin (n + 1)) R} :
```

Comment:
Let's keep the implicitness unchanged here: as a rewrite lemma it's useful to be able specify `f`.

Reply:
Not sure why I did this change. Looks like copy-pasta from `mem_support_finSuccEquiv`